In-house patch to remove unnecessary dependencies from Cinder's
requirements files. The specific reasons are as follows:

amqplib		No longer applicable
		(upstream commit e609183a39fef781aec45638c3bea4a7a6981412)

argparse	No longer applicable

lockfile	No longer applicable
		(upstream commit a497b4a214b89cb2b387f227c208cfdae62ad6b1)

netaddr		Not applicable to Solaris (VMware specific)

paramiko	Not applicable to Solaris (various drivers specific)

suds		Not applicable to Solaris (VMware specific)

wsgiref		No longer applicable

--- cinder-2013.2.3/cinder.egg-info/requires.txt.orig	2014-04-03 11:45:40.000000000 -0700
+++ cinder-2013.2.3/cinder.egg-info/requires.txt	2014-05-24 21:38:09.097874174 -0700
@@ -1,16 +1,12 @@
 pbr>=0.5.21,<1.0
-amqplib>=0.6.1
 anyjson>=0.3.3
 Babel>=1.3
 eventlet>=0.13.0
 greenlet>=0.3.2
 iso8601>=0.1.8
 kombu>=2.4.8
-lockfile>=0.8
 lxml>=2.3
-netaddr
 oslo.config>=1.2.0
-paramiko>=1.8.0
 Paste
 PasteDeploy>=1.5.0
 python-glanceclient>=0.9.0
@@ -22,6 +18,4 @@
 SQLAlchemy>=0.7.8,<=0.7.99
 sqlalchemy-migrate>=0.7.2
 stevedore>=0.10
-suds>=0.4
 WebOb>=1.2.3,<1.3
-wsgiref>=0.1.2
\ No newline at end of file
--- cinder-2013.2.3/requirements.txt.orig	2014-04-03 11:42:36.000000000 -0700
+++ cinder-2013.2.3/requirements.txt	2014-05-24 21:38:36.989936528 -0700
@@ -1,17 +1,12 @@
 pbr>=0.5.21,<1.0
-amqplib>=0.6.1
 anyjson>=0.3.3
-argparse
 Babel>=1.3
 eventlet>=0.13.0
 greenlet>=0.3.2
 iso8601>=0.1.8
 kombu>=2.4.8
-lockfile>=0.8
 lxml>=2.3
-netaddr
 oslo.config>=1.2.0
-paramiko>=1.8.0
 Paste
 PasteDeploy>=1.5.0
 python-glanceclient>=0.9.0
@@ -23,6 +18,4 @@
 SQLAlchemy>=0.7.8,<=0.7.99
 sqlalchemy-migrate>=0.7.2
 stevedore>=0.10
-suds>=0.4
 WebOb>=1.2.3,<1.3
-wsgiref>=0.1.2
